fixtures/install/playground/superface/types/starwars/character-information.ts

```diff
@@ -1,15 +1,13 @@
 import { typeHelper, TypedProfile } from '@superfaceai/one-sdk';
-/** Starwars **/
```

Comment:
Where did the doc comments go?

Reply:
the be borked :disappointed:

Reply:
fixed. they went down to properer place now